.swiftlint.yml

```diff
@@ -0,0 +1,14 @@
+# Set max line length before warning (default is 120)
+line_length: 245
+
+disabled_rules:
+  - trailing_whitespace # Disables SwiftLint complaining about whitespace characters on empty lines
+  - todo # Disables auto-warning of TODO statements
+
+identifier_name:
+  excluded:
```

Comment:
Are these exclusions needed for this project? If not, let's go ahead and clean them up.

Reply:
I removed all except for the trailing whitespace rule since Xcode adds those by default.